multimedia/gst1-libav/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gst1-libav
 PKG_VERSION:=1.15.2
-PKG_RELEASE:=2
+PKG_RELEASE:=3
```

Comment:
I think we can leave it at 2, since the only affected platform is not building it anyway; we'll save some bandwidth by not having unneeded upgrade downloads.